src/lsst/cm/prod/configs/HSC/test/hsc_bps_template.sh

```diff
@@ -11,8 +11,7 @@ setup lsst_distrib
 
 # setup PanDA env.
 latest_panda=$(ls -td /cvmfs/sw.lsst.eu/linux-x86_64/panda_env/v* | head -1)
-source ${latest_panda}/setup_lsst.sh ${WEEKLY}
```

Comment:
Did you really want to drop the setup_lsst.sh line? There is nothing in setup_panda_usdf.sh that replaces it.

Reply:
Ah, I see the comments in the Jira ticket now. I guess the only question is whether one can always be confident that the calling party will have already run the setup step.

Reply:
Yeah, I assume since it's equivalent to the above few lines 
```# setup LSST env.
export WEEKLY='{lsst_version}'
source /cvmfs/sw.lsst.eu/linux-x86_64/lsst_distrib/${WEEKLY}/loadLSST.bash
setup lsst_distrib
```
it's okay.

Reply:
On the other hand, if they have run it already, you would undo whatever setup they had because you've hard-coded it to look for the latest version, which may or may not be what they set up. So I'd agree with dropping it, but it might be a good idea to add a comment saying that it won't run if the rest of the stack isn't set up elsewhere (which in practice it almost certainly always is).

Reply:
Done!